cmake/ProjectsTagsMaster.cmake

```diff
@@ -2,4 +2,5 @@ macro(set_tag tag_name tag_value)
     if(NOT ${tag_name})
         set(${tag_name} ${tag_value})
     endif()
-endmacro()
\ No newline at end of file
+endmacro()
+set(OsqpEigen_TAG 9ccbfd1f91285670514af6a9af3775b7f7736a67) 
```

Comment:
Instead of using the commit we can add the tag [`v0.4.1`](https://github.com/robotology/osqp-eigen/releases/tag/v0.4.1)

Reply:
Fixed in https://github.com/robotology/robotology-superbuild/pull/260/commits/3e886f088c752d01265b229355b10640f14cbbcc . When the CI finishes, I will merge, thanks.

Reply:
Ping me if I forgot to merge.